editor/plugins/node_3d_editor_plugin.h

```diff
@@ -349,6 +349,15 @@ class Node3DEditorViewport : public Control {
 		Variant gizmo_initial_value;
 		bool original_local;
 		bool instant;
+
+		// Numeric blender-style transforms (e.g. 'g5x').
+		// numeric_input tracks the current input value, e.g. 1.23.
+		// numeric_negate indicates whether '-' has been pressed to negate the value
+		// while numeric_next_decimal is 0, numbers are input before the decimal point
+		// after pressing '.', numeric next decimal changes to -1, and decrements after each press.
```

Comment:
It's always -1.

Reply:
It's decremented here: https://github.com/godotengine/godot/pull/58389/files/b7ea61bcb90b681f7f9624cfc290f347a259309f#diff-57832e09092105245d5eaa89d7834a30f5c6bd6e5b5166450fde12f61a585d4cR2159

I can change the `--` to a `-= 1` on the next line if that would be more clear.

Reply:
It's fine, I just missed it. I guess it wouldn't be working otherwise 🤔